### DISCUSSION TITLE
Welcome to org-transclusion Discussions!

### DISCUSSION THREAD

**Discussion by nobiot** [nobiot/org-transclusion#45] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by ferrinm** [comment]
Hello, not sure if this has been documented anywhere but I just stumbled on a quick workaround to get transclusion to work on indented lines. It's really useful for me to transclude bits of code to another org file where I'm writing step-by-step instructions to replicate what I did, so keeping the transclusions indented allows ordered lists to behave properly in org-mode. The trick is to just wrap the transclusion link in a notes block (`#+begin_notes org` ... `#+end_notes`) so that the beginning of the block is at the proper indentation level, but the transcluded content itself is non-indented within the block. 

Cheers and thanks for the awesome package!

**Reply by nobiot** [reply]
@ferrinm No, it is not documented anywhere -- so cool!! Thank you. And thank you for your kind words. My life does not allow me to come back to enhancing this package yet, but I will, in April/May this year, hopefully.

Now... Would you mind sharing a screen grab of the work around by any chance? It really sounds like a great piece of knowledge, and I would love to see it myself. Also I am super curious about how you use it for your "step-by-step instructions" -- how do you "transclude bits of code"??? -- if you don't mind, and only if you have time, of course.

Thanks again, and cheers!

**Reply by ferrinm** [reply]
Apologies for the long delay - here's a screen recording showing how wrapping a transclusion link in a `notes` block allows the indentations of items in a list to be preserved:
![org-transclusion_demo](https://user-images.githubusercontent.com/14615276/109542034-62135e00-7a79-11eb-9c6a-97211b919f05.gif)
I've mainly used this when I document what I do to configure emacs packages while experimenting with them. I like to write these steps in an ordered list so I could reproduce them in the future. My emacs config is in an org file, so I can link to the relevant headline from my notes, and transclusion really comes in handy if I tweak the contents later so they automatically propagate to my notes.

**Reply by nobiot** [reply]
Thank you!
